### PR TITLE
Adding OMNI Data Import

### PR DESCRIPTION
**Pull request by abotiamnot**
Low resolution OMNI Data Import

### CODE REVIEW

File: heliopy/data/omni.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+             'AL index (Kyoto)', 'AU index (Kyoto)', 'Magnetosonic Mach Number']
+    extension = '.dat'
+    for year in range(starttime.year, endtime.year + 1):
+        fnames.append("omni2_{}".format(year))
+        dirs.append(local_dir)
+
+
+    def download_func(remote_base_url, local_base_dir,
+                      directory, fname, extension):
+        util.load()
```
Comment (abotiamnot) [inline heliopy/data/omni.py]: Need to fix the download_func! Rest is working ( I think )

File: heliopy/data/omni.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+    def processing_func():
+        thisdata = pandas.read_table(<>, names=names, delim_whitespace=True)
```
Comment (abotiamnot) [inline heliopy/data/omni.py]: Need to replace the <> with whatever filename it will be

**Comment by dstansby** [comment]
👍  Once all this works, could you add a simple plotting example to the examples folder too?

File: heliopy/data/omni.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                         ('Na/Np', u.dimensionless_unscaled),
+                         ('Flow Pressure', u.nPa),
+                         ('sigma T', u.K),
+                         ('sigma N', u.cm**-3),
+                         ('sigma V', u.km / u.s),
+                         ('sigma phi V', u.deg),
+                         ('sigma theta V', u.deg),
+                         ('sigma Na/Np', u.dimensionless_unscaled),
+                         ('Electric Field', u.mV / u.m),
+                         ('Plasma Beta', u.K * u.cm**-3 * u.nT**-2),
```
Comment (dstansby) [inline heliopy/data/omni.py]: Plasma beta should be dimensionless

File: heliopy/data/omni.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+                         ('Alfven Mach Number', u.km * u.cm**-3 / u.nT / u.s),
```
Comment (dstansby) [inline heliopy/data/omni.py]: This should be dimensionless too

**Comment by dstansby** [comment]
This also needs an entry in `CHANGES.rst`, but otherwise looks good.

File: heliopy/data/omni.py (hunk 4 above)
Comment (abotiamnot) [reply heliopy/data/omni.py]: I'm pretty sure Alfven Mach Number must be dimensionless, but can you take a look at the README for OMNI? https://cdaweb.gsfc.nasa.gov/pub/data/omni/low_res_omni/omni2.text

Comment (dstansby) [reply heliopy/data/omni.py]: Their equation is confusing, but this should definitely be dimensionless I think.

Comment (abotiamnot) [reply heliopy/data/omni.py]: I checked it out on https://omniweb.gsfc.nasa.gov/form/dx1.html , they are dimensionless, removing them!

**Pull request merged**